commands/rank.js

```diff
@@ -96,11 +98,29 @@ ${tableBoi.getTableString()}
   }
 };
 
+function reset(client, message, channel, userId) {
+  if(client.permlevel(message) < 3) {
```

Comment:
the perm check doesnt require user id?

Reply:
I think this is the right way, I copied it from `message.js`:
```
// Get the user or member's permission level from the elevation
  const level = client.permlevel(message);
```

I think some of the perm levels need more context than just userId, and `message` has all the context needed.  You can look at `config_base.txt` to see how each permLevel is calculated